concepticondata/conceptlists/Araujo-1996-289.tsv

```diff
@@ -0,0 +1,290 @@
+Araujo-1996-289-161	161	Rabbit	1136.0	RABBIT	Hase	coelho
+Araujo-1996-289-162	162	Capybara	314.0	CAPYBARA	Capivara	capivara
+Araujo-1996-289-163	163	Paca	316.0	PACA	Paca	paca
+Araujo-1996-289-164	164	Agouti	313.0	AGOUTI	Cutia	cotia
+Araujo-1996-289-165	165	Ox	1169.0	OX	Rinde	boi
+Araujo-1996-289-166	166	Pecari Tajaçu			Tayaçú	taiaçu
+Araujo-1996-289-167	167	Pecari Taitetu			Taitetú	taitetu
+Araujo-1996-289-168	168	Pig	1337.0	PIG	Hausschwein	porco doméstico
+Araujo-1996-289-169	169	Horse	615.0	HORSE	Pferd	cavalo
+Araujo-1996-289-170	170	Tapir	312.0	TAPIR	Tapir	anta
+Araujo-1996-289-171	171	Giant Anteater (Myrmecophaga Tridactyla)	181.0	ANTEATER	Tamanduá bandeira	tamanduá bandeira
+Araujo-1996-289-172	172	Southern Tamandua (Tamandua Tetradactyla)			Mixilla	melete
+Araujo-1996-289-173	173	Armadillo	1885.0	ARMADILLO	Tatu	tatu
+Araujo-1996-289-174	174	Six-Banded Armadillo (Euphractus Sexcinctus)			Tatu peba	tatu peba
+Araujo-1996-289-175	175	Six-Banded Armadillo (Euphractus Sexcinctus)			Tatu canastra	tatu canastra
+Araujo-1996-289-176	176	Sloth	2446.0	SLOTH	Faultier	bicho-preguiça
+Araujo-1996-289-177	177	Opossum	309.0	OPOSSUM	Beutelratte	gambá
+Araujo-1996-289-178	178	Bird	937.0	BIRD	Vogel	ave
+Araujo-1996-289-179	179	Egg	744.0	EGG	Ei	ovo
+Araujo-1996-289-180	180	Harpy			Harpye	harpia
+Araujo-1996-289-181	181	Hawk	307.0	HAWK	Falke	gavião
+Araujo-1996-289-182	182	Urubu (Cathartidae)	1185.0	VULTURE	Aasgeier	urubu
+Araujo-1996-289-183	183	Urubu-Rei (Sarcoramphus Papa)			Königsgeier	urubu-rei
+Araujo-1996-289-184	184	Owl	735.0	OWL	Eule	coruja
+Araujo-1996-289-185	185	Macaw	1630.0	MACAW	Arara	arara
+Araujo-1996-289-186	186	Parrot	882.0	PARROT	Papagai	papagaio
+Araujo-1996-289-187	187	Woodpecker	2447.0	WOODPECKER	Specht	pica-pau
+Araujo-1996-289-188	188	Sabia-Laranjeira (Turdus Rufiventris)			Drossel	sabiá
+Araujo-1996-289-189	189	Yellow-Rumped Cacique (Cacicus Cela)			Japiim	japim
+Araujo-1996-289-190	190	Crested Oropendola (Psarocolius Decumanus)			Japu	japu
+Araujo-1996-289-191	191	Ruddy Ground Dove (Columbina Talpacoti)			Juruty	juruti
+Araujo-1996-289-192	192	Chicken	1318.0	CHICKEN	Huhn	galinha
+Araujo-1996-289-193	193	Guan	437.0	GUAN	Jacu	jacu
+Araujo-1996-289-194	194	Black-Fronted Pipin Guan (Aburria Jacutinga)			Jacutinga	jacutinga
+Araujo-1996-289-195	195	Curassow	315.0	CURASSOW	Mutum	mutum
+Araujo-1996-289-196	196	Horned Screamer (Anhima Cornuta)			Anhuma	anhuma
+Araujo-1996-289-197	197	Wattled Jacana (Jacana Jacana)			Jaçanã	jaçanã
+Araujo-1996-289-198	198	Duck	1360.0	DUCK	Ente	pato
+Araujo-1996-289-199	199	Heron	758.0	HERON	Reiher	garça
+Araujo-1996-289-200	200	Aquatic Bird			?Wasservogel	ave aquática
+Araujo-1996-289-201	201	Caiman	2438.0	CAIMAN	Jacaré	jacaré
+Araujo-1996-289-202	202	Yacare Caiman			Jaboty	jabuti
+Araujo-1996-289-203	203	Snake	730.0	SNAKE	Schlange	cobra
+Araujo-1996-289-204	204	Anaconda (Water Boa)	896.0	ANACONDA (WATER BOA)	Giboya	jibóia
+Araujo-1996-289-205	205	Jararaca (Bothrops Insularis)	2654.0	VIPER	Jararaca	jararaca
```

Comment:
Unmap?

Reply:
Yes. In general, we do not do fuzzy mapping with VIPER vs. a specific kind of viper with a latin name, etc.